src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/dec/BrotliDecoder.cs

```diff
@@ -39,6 +41,19 @@ private void EnsureNotDisposed()
                 throw new ObjectDisposedException(nameof(BrotliDecoder), SR.BrotliDecoder_Disposed);
         }
 
+        /// <summary>Decompresses data that was compressed using the Brotli algorithm.</summary>
+        /// <param name="source">A buffer containing the compressed data.</param>
+        /// <param name="destination">When this method returns, a byte span containing the decompressed data.</param>
+        /// <param name="bytesConsumed">The total number of bytes that were read from <paramref name="source" />.</param>
+        /// <param name="bytesWritten">The total number of bytes that were written in the <paramref name="destination" />.</param>
+        /// <returns>One of the enumeration values that indicates the status of the decompression operation.</returns>
+        /// <remarks><![CDATA[
+        /// The return value can be as follows:
```

Comment:
If the tool closed the xref tags could we lose the ugly CDATA?

Reply:
Also, doesn't VS/the .NET world expect these to be `<see cref="System.Buffers.OperationStatus.Done"/>`? How can we satisfy those tools/VS and also the docs tools that want xref?

Reply:
See my other [comment](https://github.com/dotnet/runtime/pull/46716/files#r554242905).